addons/nfsserverprovisioner/2.3.x/nfsserverprovisioner-1.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  annotations:
+    appversion.kubeaddons.mesosphere.io/nfs-server-provisioner: 2.3.0
+    catalog.kubeaddons.mesosphere.io/addon-revision: 2.3.0-1
+  labels:
+    kubeaddons.mesosphere.io/name: nfsserverprovisioner
+  name: nfsserverprovisioner
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: false
+    - name: azure
+      enabled: false
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: false
+  chartReference:
+    chart: nfs-server-provisioner
+    repo: https://mesosphere.github.io/charts/staging
+    version: 0.6.0
+    values: |
+      persistence:
+        enabled: true
+        size: 100Gi
+      storageClass:
+        name: nfsserverprovisioner
+      resources:
+        limits:
+           cpu: 1000m
```

Comment:
We should not set limits to the storage solution, you don't want to stop working because it needs more resources than it has.